pgtoolkit/hba.py

```diff
@@ -260,26 +244,6 @@ def auth_options(self) -> list[tuple[str, str]]:
             (f, getattr(self, f)) for f in self.fields if f not in self.COMMON_FIELDS
         ]
 
-    @property
-    def database(self) -> str:
```

Comment:
We could avoid the breaking change by defining `databases` and `users` properties that would to the `.split(",")`, no?